webview/src/shared/variables.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable scss/dollar-variable-empty-line-before */
```

Comment:
Maybe we could add an exception to `after-comment` (https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/dollar-variable-empty-line-before/README.md#after-comment-1) and just break this file with comments. We have sizes mixed in between colors... I would also turn on the exception for `after-dollar-variable` (https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/dollar-variable-empty-line-before/README.md#after-dollar-variable) to remove inconsistent breaks in these variables.

Reply:
Fixed!